es/es-repository-s3/src/main/java/org/elasticsearch/repositories/s3/S3Repository.java

```diff
@@ -126,9 +127,9 @@
 
     /**
      * When set to true metadata files are stored in compressed format. This setting doesn’t affect index
-     * files that are already compressed by default. Defaults to false.
+     * files that are already compressed by default. Defaults to true.
      */
-    static final Setting<Boolean> COMPRESS_SETTING = Setting.boolSetting("compress", false);
+    static final Setting<Boolean> COMPRESS_SETTING = Setting.boolSetting("compress", true); // TODO: in ES, it defaults to false!
```

Comment:
why do we want to change the defaults in a cleanup commit? please elaborate the TODO (what to do)

Reply:
please see #8636 - the default is also changed in upstream - will remove all todos